util/build-gnu.sh

```diff
@@ -138,8 +152,17 @@ sed -i -e '/tests\/misc\/invalid-opt.pl/ D' \
 sed -i -e '/tests\/misc\/seq-precision.sh/ D' \
     Makefile
 
+# TODO: many files and some directories modified with 'sed' in the sections below either no longer exist or have been moved
+# TODO: Might need to review and updated sections below
+# TODO: As a result this script will fail when executed normally as 'bash util/build-gnu.hs' due to the 'set -e' set at the beginning 
+# TODO: The rest of the 'sed' commands after first failure in this scenario will not be executed as bash will exit on first error
+# TODO: However, the behaviour might be different when running it via GitHub actions (GnuTests)
```

Comment:
Yes, this has been changed after the last coreutils release. You should be able to fix it by checking out the tag of the latest release.

Reply:
interesting ... I have branched of from main that was already at release 0.0.20, admittedly not from the release tag, just HEAD on main:
Cargo.toml
[package]
name = "coreutils"
version = "0.0.20"

has there been a regression since that release (it looks like it is latest)?

Reply:
just checked out release by tag 0.0.20 ... the issues are still there starting at line 135 on `build-gnu.sh`
i.e. the script will stop executing here:
`sed -i '/INT_OFLOW/ D' tests/misc/printf.sh`

since `tests/misc/printf.sh` does not exist at main (HEAD) GNU coreutils at 'https://github.com/coreutils/coreutils.git'

what am I missing?

Reply:
ok, never mind :) what I am missing is that you were talking about checking out latest release tag of GNU coreutils, not this repo. Will update the PR.

On that note: would it make sense to modify the message to indicate that latest release tag needs to be checkout after cloning GNU coreutils?
`
Could not find GNU coreutils (expected at '<mypath>/gnu')
Run the following to download into the expected path:
git clone --recurse-submodules https://github.com/coreutils/coreutils.git "<mypath>/gnu"
`

Reply:
Updated PR to target correct GNU coreutlis release

Reply:
Yeah I meant GNU. Sorry gor the confusion 😄

Reply:
yep, thanks! Added GNU coreutlis release tag to the script